packages/cuba-react-ui/src/ui/table/DataTableCell.tsx

```diff
@@ -20,16 +21,23 @@ export const DataTableCell = (props: DataTableCellProps): ReactNode => {
     );
   } else if (props.propertyInfo.attributeType === 'ENUM') {
     return (
```

Comment:
If you are using 'return' in each else-if block, you can not to use else-if but use set of 'if' statements. This code will be easier to read.

Reply:
done